src/audit/secrets_outside_environment.rs

```diff
@@ -0,0 +1,83 @@
+use github_actions_models::{
+    common::{Env, EnvValue, expr::LoE},
+    workflow::job::StepBody,
+};
+
+use super::{Audit, audit_meta};
+use crate::{finding::Confidence, models::StepCommon, AuditState, AuditLoadError, Persona};
+
+pub(crate) struct SecretsOutsideEnvironment;
+
+audit_meta!(
+    SecretsOutsideEnvironment,
+    "secrets-outside-environment",
```

Comment:
I'm trying to make new audit names a bit shorter, so some ideas here:

1. `secret-outside-env`
2. `secret-without-env`
3. `secret-missing-env`
4. `secret-no-env` (shortest, but maybe not easy to understand)

Curious if you have any other naming thoughts as well :slightly_smiling_face:

Reply:
Picked number 2 and fixed in `96c90ac`.